src/python_inspector/setup_py_live_eval.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Requirements-Builder
+# Copyright (C) 2015, 2016, 2017, 2018 CERN.
+#
+# Requirements-Builder is free software; you can redistribute it and/or
+# modify it under the terms of the Revised BSD License; see LICENSE
+# file for more details.
+#
+"""Generate requirements from `setup.py` and `requirements-devel.txt`."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+import re
+import sys
+
+try:
+    import configparser
+except ImportError:  # pragma: no cover
+    import ConfigParser as configparser
+
+import mock
+import pkg_resources
+import setuptools
+
+
+def parse_set(string):
+    """Parse set from comma separated string."""
+    string = string.strip()
+    if string:
+        return set(string.split(","))
+    else:
+        return set()
+
+
+def minver_error(pkg_name):
+    """Report error about missing minimum version constraint and exit."""
+    print(
+        'ERROR: specify minimal version of "{0}" using ' '">=" or "=="'.format(pkg_name),
+        file=sys.stderr,
+    )
+    sys.exit(1)
+
+
+def build_pkg_name(pkg):
+    """Build package name, including extras if present."""
+    if pkg.extras:
+        return "{0}[{1}]".format(pkg.project_name, ",".join(sorted(pkg.extras)))
+    return pkg.project_name
+
+
+def parse_pip_file(path):
```

Comment:
You should remove this function that we do not use and is not as good as pip-requirements-parser.